config/sync/views.view.stanford_opportunities.yml

```diff
@@ -0,0 +1,7737 @@
+uuid: e8590ccb-d492-42e7-b2a0-3e87986f3ab2
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.stanford_card
+    - core.entity_view_mode.node.stanford_h3_card
+    - field.storage.node.su_event_alt_loc
+    - field.storage.node.su_event_date_time
```

Comment:
Looks like you have alot of event stuff mixed in here. There's even a "Schedule" display intended for events, some event categories, event fields, etc.

To remove some of those, you might have to enable the "Default" display on views: `drush cset views.settings ui.show.default_display 1 -y`
This will show the "Default" display when editing views.
![image](https://github.com/user-attachments/assets/7b8e83f1-2855-4071-9d47-57deaf880cd1)